website/docs/cheatsheets/python/v1/troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Troubleshooting
+id: troubleshooting
+description: A cheat sheet for Azure ML.
+keywords:
+  - azure machine learning
+  - aml
+  - troubleshooting
+---
+
+### Error: az acr login- APIVersion 2020-11-01-preview is not available. 
+**Description**
+I built a mlnet image successfully using Azure Machine Learning compute instance last week. However, I got the error when rerun the same notebook. 
+
+The command I run:
+```python
```

Comment:
This doesn't look like python. You can use `bash`, or just leave it blank.

Similarly for the snippet(s) below.

Reply:
And can you remove the leading "!", which presumably was there because you ran this in a notebook?